monkey/monkey_island/cc/ui/src/components/report-components/security/StolenPasswords.js

```diff
@@ -22,11 +22,16 @@ class StolenPasswordsComponent extends React.Component {
   render() {
     let defaultPageSize = this.props.data.length > pageSize ? pageSize : this.props.data.length;
     let showPagination = this.props.data.length > pageSize;
+    let table_data = this.props.data;
+    if(this.props.format) {
+      table_data = getCredentialsTableData(this.props.data);
+    }
```

Comment:
The commit message says this logic was only added because the ATT&CK report uses it which gets its credentials' data from the telemetry which is already formatted. I'm assuming we're planning on reworking that when we fix the telemetry? Consider adding a comment here about it.

Reply:
Exactly. 
I will add a note. Thanks!